scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulerFrontend.java

```diff
@@ -409,6 +407,31 @@ public TaskResult getTaskResult(String jobId, String taskName) throws NotConnect
         return this.getTaskResult(JobIdImpl.makeJobId(jobId), taskName);
     }
 
+
+    @Override
+    public List<TaskResult> getTaskResultByTag(JobId jobId, String taskTag)
```

Comment:
With this implementation, we rely on the getTaskResultFromIncarnation method which loads task results one by one. As we want to achieve a multiple load, this is not optimal as each time a DB request will be run.

It would be faster to add in SchedulerDBManager a new method able to load multiple task results at once. 

But as I look at the code, I see the "index" parameter can be a problem (to be honest I don't really understand it).

If tags are stored in the DB, it would be possible to run a query which selects tasks by tags.